content/docs/atom-guide/look-dev/materials/materials.md

```diff
@@ -0,0 +1,79 @@
+
+<!-- SVG file edited using https://app.diagrams.net/ -->
+![Material Files Diagram](/images/atom-guide/materials/material-file-diagram.svg)
+
+Materials and material types are stored as data items in JSON files. The Atom Material Builder converts the data files into material assets. The material assets are then consumed by the application and applied to a mesh's surface. 
+
+## Materials
+A **material** is a data item that references a material type and defines the material's property values. The material must reference a material type, which defines how the material works and what properties are available. Alternatively, a material might reference another *parent* material and inherit its property values. The material inheritance tree can go as deep as needed. 
+
+Material files (`*.material`) are in JSON format. They can be edited using the Material Editor or authored directly. The file is in a simple JSON format and can be authored or machine-generated by scripting outside the Material Editor.
+
+For more information, see [Material File Specification](/docs/atom-guide/look-dev/materials/material-file-spec/).  
+
+### Material File Example
+
+In the following example, the material uses the **StandardPBR** material type and contains the properties `baseColor`, `roughness`, and `normal`. 
+
+```json
+{
+    "materialType": "Materials/Types/StandardPBR.materialtype",
+    "materialTypeVersion": 6,
+    "propertyValues": {
+        "baseColor.textureMap": "Textures/Default/default_basecolor.tif",
+        "roughness.textureMap": "Textures/Default/default_roughness.tif",
+        "normal.factor": 0.8,
+        "normal.textureMap": "Textures/Default/default_normal.tif"
+    }
+}
+```
+
+{{< image-width "/images/atom-guide/materials/simple-example.jpg" "720" "Example Material Screenshot" >}}
+
+{{< note >}}
+The textures in the sample above can be found in `Gems/Atom/Feature/Common/Assets/Textures/Default`.
+{{< /note >}}
+
+## Material Types
+A **material type** is a data item that includes everything that's needed to describe how to render a mesh:  
+- A set of *material property* definitions
+- Links to shaders
+- Scripts that describe how to consume the material properties
+- Metadata for the Material Editor to use when displaying material properties
+
+Material type files (`*.materialtype`) are in JSON format and are authored directly. This will link together several other files as well, to form a complete material type definition. These additional files include *.shader*, *.azsl*, and *.lua* files.
+
+{{< note >}}
+A node graph based tool for authoring material types is in development, but until that's available material types are authored directly.
+{{< /note >}}
+
+For more information, see [Material Type Authoring Guide](/docs/atom-guide/dev-guide/materials/material-type-authoring-guide/).  
+
```

Comment:
- Move note to the bottom of this section. Callout boxes are visual interruptions when reading, so I try to move them to the end, as long as the callout still makes sense there. 
- For sentence in note, split into two sentences. 
```suggestion

For more information, see [Material Type Authoring Guide](/docs/atom-guide/dev-guide/materials/material-type-authoring-guide/).  

{{< note >}}
A node graph based tool for authoring material types is in development. Until that's available, material type are authored directly.
{{< /note >}}

```